test/comment.e2e-spec.ts

```diff
@@ -432,40 +440,41 @@ describe('comments Module', () => {
       expect(respond.body.message).toBe('Details are as follows');
       expect(respond.status).toBe(200);
       expect(respond.body.code).toBe(200);
-      expect(respond.body.data.id).toBeDefined();
-      expect(respond.body.data.commentableId).toBe(questionIds[0]);
-      expect(respond.body.data.commentableType).toBe('question');
-      expect(respond.body.data.content).toContain('zfgg好帅');
-      expect(respond.body.data.createdAt).toBeDefined();
-      expect(respond.body.data.disagreeCount).toBe(1);
-      expect(respond.body.data.agreeCount).toBe(0);
-      expect(respond.body.data.agreeType).toBe('Disagreed');
+      expect(respond.body.data.comment.id).toBeDefined();
+      expect(respond.body.data.comment.commentable_id).toBe(questionIds[0]);
+      expect(respond.body.data.comment.commentable_type).toBe('QUESTION');
+      expect(respond.body.data.comment.content).toContain('zfgg好帅');
+      expect(respond.body.data.comment.user).toStrictEqual(TestUserDto);
+      expect(respond.body.data.comment.created_at).toBeDefined();
+      expect(respond.body.data.comment.disagree_count).toBe(1);
+      expect(respond.body.data.comment.agree_count).toBe(0);
+      expect(respond.body.data.comment.attitude_type).toBe('DISAGREE');
     });
-    it('should return to InvalidAgreeTypeError', async () => {
+    it('should return to InvalidAttitudeTypeError', async () => {
       const commentId = CommentIds[1];
       const respond = await request(app.getHttpServer())
-        .post(`/comments/${commentId}/attitude`)
+        .put(`/comments/${commentId}/attitude`)
         .set('Authorization', `Bearer ${TestToken}`)
-        .send({ attitudeType: '5' });
-      expect(respond.body.message).toContain('InvalidAgreeTypeError:');
+        .send({ attitude_type: 'LIKE' });
+      expect(respond.body.message).toContain('InvalidAttitudeTypeError:');
       expect(respond.status).toBe(400);
       expect(respond.body.code).toBe(400);
     });
     it('should return CommentNotFoundError', async () => {
       const commentId = 114514;
       const respond = await request(app.getHttpServer())
-        .post(`/comments/${commentId}/attitude`)
+        .put(`/comments/${commentId}/attitude`)
         .set('Authorization', `Bearer ${TestToken}`)
-        .send({ attitudeType: '2' });
+        .send({ attitude_type: '2' });
```

Comment:
Maybe you mean 'Agree'

Reply:
https://github.com/SageSeekerSociety/cheese-backend/commit/b4bd97d35965354a91af766b9ba96b15c9e877dd